sdk/servicebus/azure-servicebus/azure/servicebus/aio/_async_utils.py

```diff
@@ -39,7 +42,7 @@ def get_running_loop():
         return asyncio.get_event_loop()
 
 
-async def create_authentication(client):
+async def create_authentication(client: "ServiceBusClient"):
```

Comment:
so for [Python 3.7+, we can actually add the following line](https://docs.python.org/3/whatsnew/3.7.html#pep-563-postponed-evaluation-of-annotations) at the top:
```
from __future__ import annotations
```
 and forward reference here without the quotes:
```suggestion
async def create_authentication(client: ServiceBusClient):
```